docs/includes/file-get-started-excel-react.md

```diff
@@ -41,46 +39,219 @@ In this article, you'll walk through the process of building an Excel add-in usi
     
     After you complete the wizard, the generator will create the project and install supporting Node components.
 
-4.  Open **src/components/App.tsx**, search for the comment "Update the fill color," then change the fill color from 'yellow' to 'blue', and save the file. 
+4. Navigate to the root folder of the project.
 
-    ```js
-    range.format.fill.color = 'blue'
+    ```bash
+    cd "My Office Add-in"
+    ```
 
+## Update the code
+
+1. In your code editor, open the file **src/styles.less**, add the following styles to the end of the file, and save the file.
+
+    ```css
+    #content-header {
+        background: #2a8dd4;
+        color: #fff;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 80px; 
+        overflow: hidden;
+        font-family: Arial;
+        padding-top: 25px;
+    }
+
+    #content-main {
+        background: #fff;
+        position: fixed;
+        top: 80px;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        overflow: auto; 
+        font-family: Arial;
+    }
+
+    .padding {
+        padding: 15px;
+    }
+
+    .padding-sm {
+        padding: 4px;
+    }
+
+    .normal-button {
+        width: 80px;
+        padding: 2px;
+    }
     ```
 
-5. In the `return` block of the `render` function within **src/components/App.tsx**, update the `<Herolist>` to the code below, and save the file. 
+2. The project template that the Office Add-ins Yeoman generator created includes a React component that is not needed for this quick start. Delete the file **src/components/HeroList.tsx**.
```

Comment:
Is this a necessary step for the quick start's success? If not, maybe it should be in a note instead of as a step.

Reply:
It's not necessary for the quick start's success per se. However, we want the quick start to only contain code that's actually being used, so the user can clearly identify which components are doing what in the project.